### PR TITLE
Knoten Camera Operations Tutorial

### LINKED ISSUE #22: Sensor Utils tutorial
create a tutorial on how to use the sensor utils library in conjunction with the CSM stack.

### PR DESCRIPTION
**Pull request by amystamile-usgs**
closes #22 and #26 

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: docs/getting-started/csm-stack/knoten-camera-tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+    "This tutorial requires Knoten version 0.4 or higher.\n",
+    "\n",
+    "`conda install -c conda-forge knoten=0.4`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Create a CSM camera for a PDS image"
```
Comment (chkim-usgs) [inline docs/getting-started/csm-stack/knoten-camera-tutorial.ipynb:19]: I ran into a `matplotlib` import issue in this cell and had to install `matplotlib` separately after running the `conda install` command above in a new conda environment. Do we want to add another install step above for `matplotlib`?

**Review by chkim-usgs — COMMENTED** [review]
Looks like the CI spellcheck action is incorrectly marking something in the notebook as misspelled again, should we update *.codespellrc* to skip over `*.ipynb` extensions as well?

**Comment by amystamile-usgs** [comment]
> Looks like the CI spellcheck action is incorrectly marking something in the notebook as misspelled again, should we update _.codespellrc_ to skip over `*.ipynb` extensions as well?

Yeah I don't think it likes the image code in one of the cells.

File: docs/getting-started/csm-stack/knoten-camera-tutorial.ipynb (hunk 1 above)
Comment (amystamile-usgs) [reply docs/getting-started/csm-stack/knoten-camera-tutorial.ipynb:19]: Just took a look and matplotlib is listed in knoten's environment.yml so it should have been included in the knoten build. Not sure what happened. I'll look into this.

**Review by chkim-usgs — APPROVED** [review]

**Pull request merged**